addons/project/static/src/js/project_list.js

```diff
@@ -39,14 +39,44 @@ const ProjectListController = ListController.extend({
         return this._super(...arguments);
     },
 
-    _stopRecurrence(recurringResIds, resIds, mode) {
+    _countRecords(recurrenceIds, resIds) {
+        return this._rpc({
+            model: 'project.task',
+            method: 'read_group',
+            args: [
+                [['recurrence_id', 'in', recurrenceIds], ['id', 'not in', resIds]],
+                ['ids:count(id)'],
```

Comment:
since it is a count we could rename it. 😇 
```suggestion
                ['task_count:count(id)'],
```